eng/scripts/prepare_chat_tests.py

```diff
@@ -5,36 +5,49 @@
 

```

Comment:
Would this file be better placed in the ACS service directory?

Reply:
That's what I would have preferred if I could have used `test-resources-post.py`. Absent that, I don't want to clutter the service directory and figured it would be best to keep this in the "normal" spot under `eng/scripts`.